app/services/reports/acquisition_report_service.rb

```diff
@@ -45,6 +45,22 @@ def distributed_disposable_diapers
                                .sum('line_items.quantity')
     end
 
```

Comment:
I think we need some new wording... because the diapers from kits are still distributed...  and "total diapers distributed"  implies cloth as well.     

I don't have a *good* suggestion as yet, though.  

We could say distributed_disposable_diapers_not_from_kits,   distributed_disposable_diapers_from_kits, and total_distributed_disposable_diapers.       Not at all concise,  but clear?